README.md

```diff
@@ -1,13 +1,19 @@
-<img src="http://knodle.cc/static/logo-866f6c91f8135d66107abccbec7bf77e.png" height="100">
+<img src="img/knodle_logo_1.jpg" height="120"/>
```

Comment:
This was a change from the PyPI PR. The same README is used on the PyPI knodle webpage, where no additional files can be included, but only linked images. Therefore I replaced the file with a link. 

If not neccessary, I would only change the height, not the src.